### PR TITLE
Release Function Mesh 0.6.0

### PR DESCRIPTION
**Pull request by freeznet**
### Motivation

Release Function Mesh 0.6.0 and apply the change log.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [x] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# Function Mesh v0.6.0 Release Notes
+
+## v0.6.0 What's New
+
+- [Test] Add integration tests for Pulsar using oauth2 ([#459](https://github.com/streamnative/function-mesh/pull/459))
+- [Controller] Add support for oauth2 with Pulsar ([#463](https://github.com/streamnative/function-mesh/pull/463))
+- [Controller] Add support for log rotation ([#417](https://github.com/streamnative/function-mesh/pull/417))
+- [Controller] Rename the metrics port name to make Istio compatible ([#466](https://github.com/streamnative/function-mesh/pull/466)
+- [Controller] Improve the stability of autoscaling when HPA is enabled ([#450](https://github.com/streamnative/function-mesh/pull/450))
+- [Controller] Add support for Pulsar window function ([#460](https://github.com/streamnative/function-mesh/pull/460))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:10]:
```suggestion
- [Controller] Support Pulsar window function ([#460](https://github.com/streamnative/function-mesh/pull/460))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+- [Controller] improve FunctionMesh labels ([#451](https://github.com/streamnative/function-mesh/pull/451))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:11]:
```suggestion
- [Controller] Improve Function Mesh labels ([#451](https://github.com/streamnative/function-mesh/pull/451))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+- [Runner Images] Make runner images use different Java version based on pulsar version ([#469](https://github.com/streamnative/function-mesh/pull/469))
```
Comment (Huanli-Meng) [inline CHANGELOG.md:12]:
```suggestion
- [Runner Images] Make runner images use different Java versions based on the Pulsar versions ([#469](https://github.com/streamnative/function-mesh/pull/469))
```

**Review by Huanli-Meng — APPROVED** [review]
Overall LGTM and only some minor comments.

**Review by jiangpengcheng — APPROVED** [review]

**Comment by nlu90** [comment]
👀

**Pull request merged**